src/mpyl/steps/build/docker_build.py

```diff
@@ -133,3 +137,9 @@ def execute(self, step_input: Input) -> Output:
             message=f"Failed to build docker image for {step_input.project.name}",
             produced_artifact=None,
         )
+
+    @staticmethod
+    def substitute_pr_number(original_value: Optional[str], pr_number: Optional[int]):
```

Comment:
You might want to combine this method with `src/mpyl/steps/deploy/k8s/resources/traefik.py:38` (extract that method from the ingress creation and create a helper method to substitute PR number and/or the other wildcards). Not sure about where to put this common method though, so if you don't find a good place, maybe it's fine as is.

Reply:
I had a look, but I honestly wouldn't know where, so I'm leaving it where it is 😁

Reply:
Since Daniel is out of office this week, I can take a look tomorrow. I'll also fix the lint error then